third_party/terraform/website/docs/d/game_services_game_server_deployment_rollout.html.markdown

```diff
@@ -0,0 +1,72 @@
+---
+subcategory: "Game Servers"
+layout: "google"
+page_title: "Google: google_game_services_game_server_deployment_rollout"
+sidebar_current: "docs-google-datasource-game-services-game-server-deployment-rollout"
+description: |-
+  Get the rollout state.
+---
+
+# google\_game\_services\_game\_server\_deployment\_rollout
+
+Use this data source to get  the rollout state. 
+
+https://cloud.google.com/game-servers/docs/reference/rest/v1beta/GameServerDeploymentRollout
+
+## Example Usage 
+
+
+```hcl
+data "google_game_services_game_server_deployment_rollout" "qa" {
+    provider = google-beta
+    deployment_id = "tf-test-deployment-s8sn12jt2c"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+
+* `deployment_id` 
+  The deployment to rollout the new config to. Only 1 rollout must be associated with each deployment.
```

Comment:
This phrasing works fine when we're creating a rollout, but it's not as nice when we're reading it. Maybe something like "The deployment to get the rollout state from"